chapters/meta-information.adoc

```diff
@@ -236,3 +236,35 @@ x-gateway-upstream-targets:
 <...>
 ----
 
+[#252]
+== {MUST} provide server url
+
+Configure the API gateway server url for your resource, ensure variable are configured if templating.
```

Comment:
Can you add, 

URL is dependent on audience, external-public should be have public prepended. i.e. public.api.